route/build/src/read_remap.f90

```diff
@@ -4,101 +4,115 @@ module read_remap
 use netcdf
 use public_var
 use read_netcdf, only:get_nc
-use globalData,  only:remap_data
+use read_netcdf, only:get_nc_dim_len
+use read_netcdf, only:get_units
 
 implicit none
 
 private
 
 public::get_remap_data
+public::get_runoff_metadata
```

Comment:
This routine seems to be out of place. This routine is for runoff data not for remap. maybe move this to read_runoff.f90?